modules/ossm-rn-fixed-issues.adoc

```diff
@@ -15,7 +15,22 @@ Provide the following info for each issue if possible:
 ////
 
 The following issues have been resolved in the current release:
-//current release is 2.5.1/2.4.7/2.3.11 scheduled for April 18, 2024
+
+//current release is 2.5.2/2.4.8/2.3.12 scheduled for May 22, 2024
+
+* https://issues.redhat.com/browse/OSSM-6331[OSSM-6331] Previously, invalid `LogLevel` values were accepted in `.smcp.general.logging.componentLevels` and the `ServiceMeshControlPlane` was created. Now, an error message is shown if an invalid value is used, and the control plane is not created.
```

Comment:
The first sentence is phrased unclearly to me. Invalid value acceptance _caused_ `ServiceMeshControlPlane` resources to be created?

Reply:
`ServiceMeshControlPlane` should be followed by a noun. Likely `.scmp.general.logging.componentLevels`, too.